test/README.md

```diff
@@ -70,7 +70,7 @@ deactivate
 
 ```python
     @pytest.mark.parametrize(
-        "binary, package, version, are_in, not_in",
+        "binary, product, version, are_in, not_in",
```

Comment:
Change it to module also.

Reply:
Yeah I forgot to update the readme files